gensim/scripts/make_wiki.py

```diff
@@ -1 +0,0 @@
-make_wikicorpus.py
```

Comment:
Agreed. However, I don't know how this changed happened with the sym links. I have not touched these files, and I am happy to reverse this change, if I knew how to.